docs/reference/release-notes/highlights-7.7.0.asciidoc

```diff
@@ -18,3 +18,110 @@ pivot and summarize your data and store it in a new index. See
 {ref}//transform-apis.html[{transform-cap} APIs].
 
 // end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+=== Finer memory control for bucket aggregations
+
+A new setting has been introduced - `search.check_buckets_step_size` - to
+better control how the coordinating node will allocate memory when aggregating
+buckets. The allocation of buckets will now be done in steps, each step
+allocating a number of buckets equal to this setting. A parent circuit breaker
+check is performed on allocation, to avoid an OOM failure.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+=== A new way of searching: asynchronously
+
+Searches can now be submitted to a new endpoint, `_async_search`, that works
+exactly the same as the `_search` endpoint, except that instead of blocking and
+returning the final response only once it's entirely available, it will pan the
+results as they become available, asynchronously.
+
+The request takes a parameter, `wait_for_completion`, which controls how long
+the server will wait until it sends back a response. The first response
+contains among others a search unique ID, a response version, an indication if
+this response is partial or not, plus the usual metadata (shards involved,
+number of hits etc) and potentially results. If the response is not complete
+and final, the client can continue polling for results issuing a new request
+using the provided search ID; if new results are available, the returned
+version is incremented and the new results batch returned. This can go on until
+all the results are fetched.
+
+Unless deleted earlier by the user, the asynchronous searches are kept alive
+for a given interval. This defaults to 5 days and can be controlled by another
+request parameter, `keep_alive`.
+
+This functionality is offered under the free {xpack} Basic license.
+
```

Comment:
We typically avoid mentioning specific licenses in the docs. This keeps https://www.elastic.co/subscriptions as the universal source of truth.
```suggestion
```